biomage/utils/data.py

```diff
@@ -113,7 +112,6 @@ def copy_dynamodb_records(experiment_id, source_table, target_table, config):
 
     items_to_insert = []
     for item in items:
-        item = add_env_user_to_experiment(cfg=item)
```

Comment:
yes, but I won't be doing that because it will be completely deprecated once we move to SQL and no one is using it. I just added this changes because I needed them and didn't want to spend more time on it.